include/aos/common/crypto.hpp

```diff
@@ -333,6 +333,24 @@ inline void Encode(const Array<ObjectIdentifier>& src, Array<uint8_t>& asn1Value
     (void)asn1Value;
 }
 
+/**
+ * Encodes big integer in ASN1 format.
+ *
+ * @param number big integer.
+ * @param[out] asn1Value result ASN1 value.
```

Comment:
[al1img](https://github.com/al1img) [Dec 21, 2023](https://github.com/aoscloud/aos_core_lib_cpp/pull/49#discussion_r1433702228)

Remove empty line

add [in] [out] for params
Author
@[mykola-kobets-epam](https://github.com/mykola-kobets-epam) mykola-kobets-epam [Dec 21, 2023](https://github.com/aoscloud/aos_core_lib_cpp/pull/49#discussion_r1433918605)

done